### PR TITLE
[feed] Fix Null Pointer Exceptions and reduce log spam

### LINKED ISSUE #6906: [feed] NullPointerException if all Channels are not linked
OH and feed 2.5.1-2

Whenever the feed binding or OH is restarted, I see the following log entries. I think the The INFO entries can be set to DEBUG, though maybe it can be handled better. The WARNING is an exception and seems like there is something wrong, but everything seems to be working fine. It is annoying though.
```
2020-01-26 08:48:55.163 [INFO ] [org.openhab.binding.feed.internal.handler.FeedHandler] - Cannot update channel with ID latest-description; no data has been downloaded from the server!
2020-01-26 08:48:55.164 [INFO ] [org.openhab.binding.feed.internal.handler.FeedHandler] - Cannot update channel with ID author; no data has been downloaded from the server!
2020-01-26 08:48:55.164 [INFO ] [org.openhab.binding.feed.internal.handler.FeedHandler] - Cannot update channel with ID latest-date; no data has been downloaded from the server!
2020-01-26 08:48:55.164 [INFO ] [org.openhab.binding.feed.internal.handler.FeedHandler] - Cannot update channel with ID description; no data has been downloaded from the server!
2020-01-26 08:48:55.164 [INFO ] [org.openhab.binding.feed.internal.handler.FeedHandler] - Cannot update channel with ID last-update; no data has been downloaded from the server!
2020-01-26 08:48:55.164 [INFO ] [org.openhab.binding.feed.internal.handler.FeedHandler] - Cannot update channel with ID title; no data has been downloaded from the server!
2020-01-26 08:48:55.164 [INFO ] [org.openhab.binding.feed.internal.handler.FeedHandler] - Cannot update channel with ID number-of-entries; no data has been downloaded from the server!
2020-01-26 08:50:55.510 [WARN ] [org.eclipse.smarthome.core.internal.common.WrappedScheduledExecutorService] - Scheduled runnable ended with an exception: 
java.lang.NullPointerException: null
        at org.openhab.binding.feed.internal.handler.FeedHandler.publishChannelIfLinked(FeedHandler.java:142) ~[?:?]
        at org.openhab.binding.feed.internal.handler.FeedHandler.refreshFeedState(FeedHandler.java:117) ~[?:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_232]
        at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[?:1.8.0_232]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_232]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[?:1.8.0_232]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_232]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_232]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_232]
```

### PR DESCRIPTION
**Pull request by 9037568**
Reduce the log-level of messages emitted to eliminate log spam.
Fix the Null Pointer Exception that occurs when various attributes have no value.

[Test jar is available](https://drive.google.com/open?id=1twN7zHn7hiAYDnMYHWd-IEeCRnV7dC5B).

Fixes #6906.

Signed-off-by: 9037568 <namraccr@gmail.com>

### CODE REVIEW

**Comment by openhab-5iver** [comment]
Tested and appears to work... thank you again, Chris!

**Comment by 9037568** [comment]
You're welcome!

File: bundles/org.openhab.binding.feed/src/main/java/org/openhab/binding/feed/internal/handler/FeedHandler.java [hunk 1]
```diff
@@ -124,33 +125,48 @@ private void publishChannelIfLinked(ChannelUID channelUID) {
...
+        SyndEntry latestEntry = getLatestEntry(currentFeedState);
+
         switch (channelID) {
             case CHANNEL_LATEST_TITLE:
-                String title = getLatestEntry(currentFeedState).getTitle();
-                state = new StringType(getValueSafely(title));
+                if (latestEntry == null || latestEntry.getTitle() == null) {
+                    state = UnDefType.UNDEF;
+                }
+                else {
```
Comment (J-N-K) [inline bundles/org.openhab.binding.feed/src/main/java/org/openhab/binding/feed/internal/handler/FeedHandler.java]: Please apply the formatter

Comment (9037568) [reply bundles/org.openhab.binding.feed/src/main/java/org/openhab/binding/feed/internal/handler/FeedHandler.java]: Done

**Comment by martinvw** [comment]
Thanks @9037568

**Pull request merged**